src/List.test.jsx

```diff
@@ -0,0 +1,40 @@
+import { render } from '@testing-library/react';
+
+import List from './List';
+
+describe('List', () => {
+  it('with empty tasks', () => {
+    const emptyTasks = [];
+
+    const { container } = render((
+      <List
+        tasks={emptyTasks}
+      />
+    ));
+
+    expect(container).toHaveTextContent('할 일이 없어요!');
+  });
+
+  test('with full tasks', () => {
```

Comment:
위에서 `it`를 썼으니 여기도 `it`로 써주는게 좋을 것 같아요
```suggestion
  it('with full tasks', () => {
```

Reply:
네네 통일했습니다.